packages/react-dom/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -975,7 +975,6 @@ function pushStartTextArea(
         children.length <= 1,
         '<textarea> can only have at most one child.',
       );
```

Comment:
```suggestion
      );
      // Fall through. `'' + children` is same as `'' + children[0]` for 1-element arrays.
```
This behavior is weird enough that it probably deserves a comment explaining what's going on.

Reply:
Added comment